### PR TITLE
[Monitoring][issue-52] Added PercentOfNull check / added ReduceMixin to checks

### PR DESCRIPTION
**Pull request by yromanyshyn**
resolves monitoring [issue-52](https://github.com/deepchecks/private-issues/issues/52)

### CODE REVIEW

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        Returns
+        -------
+        Dict[str, float]
+            percent of nulls in each columns
+        """
+        return check_result.value
+
+    def add_condition_percent_of_nulls_not_greater_than(
+        self: TPercentOfNulls,
+        threshold: float = 0.0
```
Comment (Nadav-Barak) [inline deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:84]: Would change default to 0.05

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+from deepchecks.utils.strings import format_list, format_percent
+from deepchecks.utils.typing import Hashable
+
+__all__ = ['PercentOfNulls']
+
+
+TPercentOfNulls = t.TypeVar('TPercentOfNulls', bound='PercentOfNulls')
+
+
+class PercentOfNulls(SingleDatasetCheck, ReduceMixin):
```
Comment (Nadav-Barak) [inline deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:29]: Do we need a plot example for this? @shir22 @noamzbr

**Review by Nadav-Barak — COMMENTED** [review]
Looks good! main issue is whether we need a plot example for the new check

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        l = len(data)
+        n_of_nulls = [data[c].isna().sum() for c in data.columns]
+        percent_of_nulls = [it / l for it in n_of_nulls]
+
+        display = (
+            [px.bar(x=data.columns, y=percent_of_nulls, title='Percent Of Nulls', range_y=(0, 1))]
+            if sum(n_of_nulls) > 0
+            else None
+        )
+        return CheckResult(
```
Comment (Nadav-Barak) [inline deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:67]: add             header='PercentOfNulls',

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py (hunk 1 above)
Comment (yromanyshyn) [reply deepchecks/tabular/checks/data_integrity/percent_of_nulls.py]: fixed

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py (hunk 3 above)
Comment (yromanyshyn) [reply deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:67]: fixed

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py (hunk 2 above)
Comment (yromanyshyn) [reply deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:29]: added

**Review by Nadav-Barak — APPROVED** [review]
Commented on plot example

File: docs/source/checks/tabular/data_integrity/plot_percent_of_nulls.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_percent_of_nulls:
+
+Percent Of Nulls
```
[the remark this answers is not in the document]
Comment (yromanyshyn) [reply docs/source/checks/tabular/data_integrity/plot_percent_of_nulls.py:5]: fixed

File: deepchecks/tabular/checks/data_integrity/percent_of_nulls.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        """Run check logic."""
+        dataset = context.get_data_by_kind(dataset_kind)
+        dataset = dataset.select(self.columns, self.ignore_columns, keep_label=False)
+        data = dataset.features_columns
+
+        l = len(data)
+        n_of_nulls = [data[c].isna().sum() for c in data.columns]
+        percent_of_nulls = [it / l for it in n_of_nulls]
+
+        display = (
```
[the remark this answers is not in the document]
Comment (yromanyshyn) [reply deepchecks/tabular/checks/data_integrity/percent_of_nulls.py:62]: fixed

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**